lib/report_runner.rb

```diff
@@ -34,14 +36,28 @@ def generate_report
       ga_profile_id: @ga_profile_id ,
       start_date: @start_date,
       end_date: @end_date,
-      output: @output
+      output: @output,
+      google_parent_id: @google_parent_id
     })
 
     report_generator.generate_report!
   end
 
 private
 
+  def has_google_parent_id?
+    if @output == "google-sheets"
```

Comment:
This really doesn't matter, but in the interests of cutting down the number of lines, could this be the following?
```
if @output == "google-sheets" and @google_parent_id.nil?
    @errors << "requires a drive_parent_id for google-sheets output"
    return false
return true
```

Reply:
Much cleaner - yes that didn't feel good at the time of writing. Also fixed in 98164f08086d416c3ca1b6a3b483ed9056c840c2.